duck/utils/amber_inputs.py

```diff
@@ -191,7 +191,7 @@ def re_range(lst):
                 scan = j+1
                 break
         else:
-            result.append('{}-{}'.format(lst[scan], lst[j]+1))
+            result.append('{}-{}'.format(lst[scan], lst[n-1]))
```

Comment:
I was sometimes getting `UnboundLocalError: local variable 'j' referenced before assignment` on this line. I think it's because the range in the for loop could be empty and then j never gets defined.